lib/relation-definition.js

```diff
@@ -2540,8 +2539,8 @@ EmbedsMany.prototype.updateById = function (fkId, data, options, cb) {
     if (typeof cb === 'function') {
       modelInstance.updateAttribute(propertyName,
         embeddedList, options, function(err) {
```

Comment:
I don't mind leaving whitespace/formatting/indenting till we put everything through the linter. It'll be much easier to fix those when all the other rules clean up the majority of the code. Technically, if you can't fit it on a line I assume the code should look like:

```
modelInstance.updateAttribute(propertyName, embeddedList, options,
function(err) {
  ...
});
```

However, the ROI for fixing indents/whitespace/formatting is not key for this pass. I do believe we should fix these as we see them and as we work on related features.